src/service/viewer-impl.js

```diff
@@ -143,6 +143,20 @@ export class Viewer {
     /** @const @private {!Object<string, string>} */
     this.params_ = {};
 
+    /** @private {?function()} */
```

Comment:
Ok. Mostly just thinking that this is technically not a nullable value since it's just defined two lines below. But let's keep. I think we just miss one simple construct which we can add later.